tools/pd-ctl/pdctl/command/region_command.go

```diff
@@ -278,14 +274,53 @@ func decodeKey(text string) (string, error) {
 			n = append(n, r.Next(2)...)
 			_, err := fmt.Sscanf(string(n), "%03o", &c)
 			if err != nil {
-				return "", err
+				return "", errors.WithStack(err)
 			}
 			buf = append(buf, c)
 		}
 	}
 	return string(buf), nil
 }
 
+// NewRegionsWithStartKeyCommand returns regions from startkey subcommand of regionCmd.
```

Comment:
Why?

Reply:
I do not know why `cherry-pick` will contain it,  maybe I miss it when I resolve conflict it.